idd/Energy+.idd.in

```diff
@@ -71933,11 +71933,14 @@ Boiler:HotWater,
        \type real
        \minimum> 0.0
        \default 1.0
-  A8 ; \field End-Use Subcategory
+  A8 , \field End-Use Subcategory
        \note Any text may be used here to categorize the end-uses in the ABUPS End Uses by Subcategory table.
        \type alpha
        \retaincase
        \default General
+  N10; \field Parasitic Fuel Load
+       \units W
+       \note parasitic fuel load associated with the boiler operation (i.e., standing pilot)
```

Comment:
Why does an optional field at the end of an object require transition? I added it to the end specifically so that it might not require transition, but if it does, then yes I can move it up.

The Parasitic Electric Load occurs only during on-cycle and the Parasitic Fuel Load only occurs during off-cycle, see the documentation. I agree that it's not immediately obvious that's the case based on the names, but the Parasitic Electric Load name already existed and both field names/behaviors match the Coil:Heating:Fuel object. I agree that it would be clearer if they included "On Cycle" and "Off Cycle" in their names; does that involve transition code or much extra effort? I can make the change, but am leery of getting sucked into a larger refactor.

Reply:
Oh, you're right, for some reason, I looked at the diff and decided the new field had been inserted before the end-use subcategory. 

> The Parasitic Electric Load occurs only during on-cycle and the Parasitic Fuel Load only occurs during off-cycle, see the documentation. 

That's horribly confusing. I would suggest that these fields be renamed to "On Cycle Parasitic Electric Load" and "Off Cycle Parasitic Fuel Load" both here and in Coil:Heating:Fuel. (I mean, who reads the documentation? Not me, apparently.) Renaming fields doesn't require idf transition, but it does require notes in the transition rules because it breaks epJSON inputs.

Reply:
Alright, I'll make those name changes. When you say it requires notes in the transition rules, do you mean in the e.g. RulesXX-X-X-to-XX-X-X.md file?

Reply:
> Alright, I'll make those name changes. When you say it requires notes in the transition rules, do you mean in the e.g. RulesXX-X-X-to-XX-X-X.md file?

Thanks. Yes. And I know you're trying to avoid scope creep, but do large commercial boilers really consume zero electricity during off cycle? Seems like we could use both flavors for electric parasitic.

Reply:
FYI, I updated the properties of four E+ objects to use the "On Cycle" and "Off Cycle" terminology; see [here](https://github.com/NREL/EnergyPlus/blob/boiler_parasitic_fuel/src/Transition/InputRulesFiles/Rules23-1-0-to-23-2-0.md) for the summary.